tgui/packages/tgui/interfaces/Oven.tsx

```diff
@@ -0,0 +1,218 @@
+        </Section>
+        <Section
+          title="Contents"
+          scrollable
+          buttons={
+            <Button onClick={() => act('eject_all')}>
+              <Icon name="eject" />
+            </Button>
+          }
+        >
+          {content_icons && content_icons.length ? (
+            <Stack vertical minHeight="110px" maxHeight="110px">
+              {content_icons.map((item, index) => (
+                <Stack key={index} style={{ borderBottom: '0.5px #555 solid' }}>
+                  <Stack.Item grow style={{
+                          display: 'flex',
+                        }}>
+                    <img
+                      height={26}
+                      width={26}
+                      src={`data:image/png;base64,${item}`}
+                      style={{ transform: 'translate(0, -4px)' }}
+                    />
+                    {content_names[index]}
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Button
+                      onClick={() => act('eject', { ejected_item: index + 1 })}
+                      ml={3}
+                      style={{ transform: 'translate(0, 4px)' }}
+                    >
+                      <Icon name="eject" />
+                    </Button>
+                  </Stack.Item>
+                </Stack>
+              ))}
+            </Stack>
+          ) : (
+            <Stack minHeight="110px" maxHeight="110px"><Stack.Item>(Empty)</Stack.Item></Stack>
+          )}
+        </Section>
+        {output_icon && (
+          <Section grow>
+            <Stack align>
+              <Stack.Item grow>
+                <Section title="Potential Recipe">
+                  <Stack vertical>
+                    <Stack.Item>
+                    {recipe_icons.map((item, index) => (
+                      <Stack.Item key={index} style={{ borderBottom: '0.5px #555 solid', display: "flex" }}>
+                        <img
+                          height={26}
+                          width={26}
+                          src={`data:image/png;base64,${item}`}
+                          style={{ transform: 'translate(0, -4px)' }}
+                        />
+                        {recipe_names[index]}
+                      </Stack.Item>
+                    ))}
+                    </Stack.Item>
+                    <Stack.Item>
+                      <LabeledList>
+                        <LabeledList.Item label="Cook time">
+                          {cook_time}
+                        </LabeledList.Item>
+                      </LabeledList>
+                    </Stack.Item>
+                  </Stack>
+                </Section>
+              </Stack.Item>
+              <Stack.Item><Divider vertical /></Stack.Item>
+              <Stack.Item grow>
+                <Section title="Result">
+                  <Stack vertical align="center" textAlign="center">
+                    <Stack.Item>
+                      <img
```

Comment:
This should use `Image` instead of `img` to avoid pixel scaling issues. That would let you make the icon bigger too :)